app/models/miq_request_task.rb

```diff
@@ -268,4 +268,54 @@ def valid_states
   def dialog_values
     options[:dialog] || {}
   end
+
+  def request_values
+    add_request_task_ids(self)
```

Comment:
This passing self doesn't make sense to me, since the target method is an instance method.